device/globals/player.gd

```diff
@@ -317,7 +317,7 @@ func _ready():
 		return
 
 	animation = get_node("animation")
-	vm = get_tree().get_root().get_node("vm")
+	vm = get_node("/root/vm")
```

Comment:
I think this line can be removed. `vm` is a singleton and can be accessed by just its name.